gen/gen.go

```diff
@@ -46,6 +46,8 @@ func Generate(proto *schema.WebRPCSchema, target string, config *Config) (out *G
 		WebrpcTarget     string
 		WebrpcErrors     []*schema.Error
 		Opts             map[string]interface{}
+		TmplVersion      string
+		TmplTarget       string
```

Comment:
Alternatively, we could call it "code generator" as per README.

```suggestion
		CodeGen        string // e.g. golang@v0.14.0
		CodeGenName    string // e.g. golang
		CodeGenVersion string // e.g. v0.14.0
```

```
  -target string
    	target code generator (required)
```

Reply:
I prefer this one :)